pounders/m/clean_up_before_return.m

```diff
@@ -0,0 +1,25 @@
+% This function is called to cleanup X and F, set the exit value, and display
+% reason for exiting
+
+% flag    [dbl] Termination criteria flag:
+%               = 0 normal termination because of grad,
+%               > 0 exceeded nfmax evals,   flag = norm of grad at final X
+%               = -1 if input was fatally incorrect (error message shown)
+%               = -2 model failure
+%               = -3 error from TRSP Solver or if a NaN was encountered
+%               = -4 error in TRSP Solver
+
+function [X, F, exit_flag] = clean_up_before_return(X, F, nf, exit_flag)
+
+    X = X(1:nf, :);
+    F = F(1:nf, :);
+
+    if exit_flag == -4
+        disp("An occured in minq. Exiting.");
```

Comment:
The original MINQ error message was more specific.  Aren't those details potentially useful to the user?

Reply:
Corrected!